### DISCUSSION TITLE
傻逼B站没事又开始改UI了

### DISCUSSION THREAD

**Discussion by KuiPer-LTW** [the1812/Bilibili-Evolved#4958] [功能建议 / Ideas]
![1](https://github.com/user-attachments/assets/a2417104-c502-4414-a633-af183a52e926)
哪位大手子做个改回去的功能

**Comment by Geeyun-JY3** [comment]
砍掉每个分 P 前的分 P 序数确实令人费解。

顺便关联：#4955

**Reply by Geeyun-JY3** [reply]
艹怎么连多 P 视频的分 P 跳转输入框都给砍掉了（之前的样子可参考：<https://github.com/the1812/Bilibili-Evolved/issues/1692#issuecomment-884597754>）
破站的 Web 端产品经理干什么吃的，这次更新就单纯为了统一视频合集列表和多 P 视频的视频选集的 UI 吗？

**Reply by the1812** [reply]
估计是新来的经理都不知道有这功能(

**Reply by Geeyun-JY3** [reply]
妈的还有分 P 视频的各分 P 也变成和合集一样不能通过上下文菜单在新标签页/新窗口打开和中键单击了，这更新对分 P 视频基本上是纯纯的负优化，功能一点没多，有用的特性砍了一堆。

**Comment by Geeyun-JY3** [comment]
合集的分组在播放页面的展示方式改成 tab 倒是比以前的垂直列表折叠分组体验好：
![image](https://github.com/user-attachments/assets/cc23344a-fab3-4220-8a72-98b474736ab8)
以前的样子可参考：https://github.com/the1812/Bilibili-Evolved/discussions/3764#discussioncomment-4011316

**Reply by KuiPer-LTW** [reply]
主要是字体，改了个圆润的难看的要死不说，字号还小了，纯逆天
